website/vercel.json

```
@@ -772,6 +772,19 @@
     {
       "source": "/core-concepts/writing-code",
       "destination": "/reference/appsmith-framework"
+    },
+    {
+      "source":"/v/v1.2/core-concepts/apis/taking-inputs-from-widgets",
+      "destination":"/core-concepts/connecting-to-data-sources/authentication/connect-to-apis#passing-dataparameters-to-api-calls"
+    },
+    {
+      "source": "/tutorials/building-a-store-catalog-manager/part-1-creating-a-simple-view",
+      "has": [
+        {
+          "value": "#accessing-query-results-to-a-widget"
+        }
+      ],
+      "destination": "/learning-and-resources/tutorials/building-a-store-catalog-manager/building-a-simple-ui#accessing-query-results-to-a-widget"
```

Redirects cannot handle anchors. The anchor part in a URL is _never_ sent to a server, so the server can't see it's value at all.

Besides, the `has` objects need a `type`, and has to be from (header, cookie, host or query), from https://vercel.com/docs/project-configuration#project-configuration/redirects, so, this likely won't work.

A normal redirect is all we need here. 🙂

```suggestion
    {
      "source": "/tutorials/building-a-store-catalog-manager/part-1-creating-a-simple-view",
      "destination": "/learning-and-resources/tutorials/building-a-store-catalog-manager/building-a-simple-ui"
```

---

Okay. Will raise again?